system_metrics_collector/src/topic_statistics_collector/received_message_period.hpp

```diff
@@ -113,19 +98,14 @@ class ReceivedMessagePeriodCollector : public TopicStatisticsCollector<T>
    */
   void ResetTimeLastMessageReceived()
   {
-    time_last_message_received_ = uninitialized_time_;
+    time_last_message_received_ = kUninitializedTime;
   }
 
   /**
-   * The clock to use in order to determine the rate OnMessageReceived is called.
-   */
-  rclcpp::Clock clock_;
-  /**
-   * Default uninitialized time. In order to compare rclcpp::Time they must come from
-   * the same type of clock.
+   * Default uninitialized time.
    */
-  rclcpp::Time uninitialized_time_;
-  rclcpp::Time time_last_message_received_ RCPPUTILS_TSA_GUARDED_BY(mutex_);
+  rcl_time_point_value_t time_last_message_received_{kUninitializedTime}
+  RCPPUTILS_TSA_GUARDED_BY(mutex_);
```

Comment:
`mutex_` is defined on the next line. Is that fine?

Reply:
I'm not aware of any issues with that